eng/Version.Details.xml

```diff
@@ -20,11 +20,15 @@
       <Uri>https://github.com/dotnet/cecil</Uri>
       <Sha>f7a78a77069262841fe3c3c6b89ff243fc43cefc</Sha>
     </Dependency>
-    <!-- Previous .NET Android version -->
-    <Dependency Name="Microsoft.Android.Sdk.Windows" Version="35.0.61">
+    <!-- Previous .NET Android version(s) -->
+    <Dependency Name="Microsoft.NET.Sdk.Android.Manifest-9.0.100" Version="35.0.61">
       <Uri>https://github.com/dotnet/android</Uri>
       <Sha>e7876a4f92d894b40c191a24c2b74f06d4bf4573</Sha>
     </Dependency>
+    <Dependency Name="Microsoft.NET.Sdk.Android.Manifest-8.0.100" Version="34.0.154">
```

Comment:
Do we have an idea of how many .NET 10 previews we plan to support .NET 8 targeting for? This is easy enough to revert whenever we drop that support either way, and should hopefully make dependency updating easier in general

Reply:
We will probably remove this entry around May, and remove this from the workload manifest at the same time:

https://github.com/dotnet/android/blob/e347cd37de4acc44f3a12da7b146e1140fcfc420/src/Xamarin.Android.Build.Tasks/Microsoft.NET.Sdk.Android/WorkloadManifest.in.json#L24-L33